plugins/bitrise/src/api/bitriseApi.client.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * Copyright 2021 SDA SE Open Industry Solutions
```

Comment:
not an expert in this space, but would have to guess we can't have two companies claiming copyright on the same code and both offering it as Apache 2.0 ?  may need to be one or other. note this comment applies to multiple files in this PR, but just posting it ones.

Reply:
we added the copyright with our company name and then I think the CI started "duplicating" it.